src/Modules/AuditReport/CourseAuditReport/CourseAuditReport.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { memo, useState } from 'react';
+import {
+    Box,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Paper,
+    Tabs,
+    Tab,
+} from '@mui/material';
+import { Search } from '@mui/icons-material';
+import { useQuery } from "@tanstack/react-query";
+import { getCourseNameCreateAuditReports, getCourseNameEditAuditReports } from '../../../api/commonAPI';
+import DefaultPageLayout from '../../../Components/DefaultPageLayout';
+import { AuditformatDate, CourseNameAuditReportcolumnsByTab, searchIconStyle, searchInputStyle } from '../AuditCommonCodes/auditCommonStyle';
+import DateRangeFilter from '../AuditCommonCodes/DateRangeFilter';
+
+const CourseAuditReport = () => {
+
+    const [selectedTab, setSelectedTab] = useState('Created');
+    const [searchTerm, setSearchTerm] = useState('');
+    const [fromDate, setFromDate] = useState('');
+    const [toDate, setToDate] = useState('');
+
```

Comment:
const [fromDate, setFromDate] = useState('');
    const [toDate, setToDate] = useState('');   type error const [fromDate, setFromDate] = useState(null);  or const [fromDate, setFromDate] = useState(new Date()); for both froim and to date